lib/client/bst-alexa.ts

```diff
@@ -42,7 +42,11 @@ export class BSTAlexaEvents {
 }
 
 /**
+ * @Deprecated
  * Programmatic interface for interacting with the Bespoken Alexa emulator.
+ * If you're interested in working with an Alexa Emulator for testing or other purposes, please check
```

Comment:
Double-check that the line breaks show up. I think you may need to add explicity "br" tags